### PR TITLE
release: v0.9.0

### PR DESCRIPTION
**Pull request by iGormilhit**
## Why are you opening this PR?

To finalise the v0.9.0 release.

## How to test?

- Check the changed files.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -18,6 +18,223 @@
...
+
+This release note includes the release note of ``rero-ils-ui`` version
+``v0.2.0``.
+
+User interface
+~~~~~~~~~~~~~~
+
+-  Improves the document detailed views (public and professional) with
+   tabs: a “get” tab with the holdings and information on items, a
+   “description” tab with extended document metadata and, in the public
```
Comment (pronguen) [inline RELEASE-NOTES.rst:32]:
```suggestion
   “description” tab with extended document metadata and, in the professional
```

File: RELEASE-NOTES.rst [hunk 2]
```diff
@@ -18,6 +18,223 @@
...
+   view, an “online” tab when the resource is available through an hyper
+   link. On top of these tabs, there the main metadata of the document
```
Comment (pronguen) [inline RELEASE-NOTES.rst:34]:
```suggestion
   view, an “online” tab when the resource is available through a hyperlink.
   On top of these tabs are displayed the main metadata of the document
```

File: RELEASE-NOTES.rst [hunk 3]
```diff
@@ -18,6 +18,223 @@
...
+   with the cover thumbnail.
+
+Public interface
+^^^^^^^^^^^^^^^^
+
+-  Removes the item detailed public view which is useless, all relevant
+   information being on the document detailed public view.
```
Comment (pronguen) [inline RELEASE-NOTES.rst:41]:
```suggestion
   information being on the document detailed view.
```

File: RELEASE-NOTES.rst [hunk 4]
```diff
@@ -18,6 +18,223 @@
...
+
+Professional interface
+^^^^^^^^^^^^^^^^^^^^^^
+
+-  Updates the library custom editor to normalizes button accordingly to
```
Comment (pronguen) [inline RELEASE-NOTES.rst:46]:
```suggestion
-  Updates the library custom editor to normalize buttons accordingly to
```

File: RELEASE-NOTES.rst [hunk 5]
```diff
@@ -18,6 +18,223 @@
...
+   `charter <https://github.com/rero/rero-ils/wiki/Usability-charter#buttons>`__.
+   Also, when a day is set to closed, the opening hours are hidden in
+   addition to be disabled.
+
+Search
+~~~~~~
+
+-  Moves from the ES query string, which is powerful but should not be
+   used for public search input, to ES simple query, much simpler but
+   much more resilient with syntax error in the query. It also allowed
```
Comment (pronguen) [inline RELEASE-NOTES.rst:57]:
```suggestion
   much more resilient with syntax errors in the query. It also allowed
```

File: RELEASE-NOTES.rst [hunk 6]
```diff
@@ -18,6 +18,223 @@
...
+   to set the default boolean operator to AND, which is what the
+   librarian and patrons are expecting.
```
Comment (pronguen) [inline RELEASE-NOTES.rst:59]:
```suggestion
   librarians and patrons are expecting.
```

File: RELEASE-NOTES.rst [hunk 7]
```diff
@@ -18,6 +18,223 @@
...
+   The API request are still done through the ES query string, as
+   complex query are needed to populate the user interface. A new HTTP
```
Comment (pronguen) [inline RELEASE-NOTES.rst:61]:
```suggestion
   The API requests are still done through the ES query string, as
   complex queries are needed to populate the user interface. A new HTTP
```

File: RELEASE-NOTES.rst [hunk 8]
```diff
@@ -18,6 +18,223 @@
...
+   query optional parameter is added to identify the simple query:
+   ``&simple=1``.
+-  Sets the same AND boolean operator instead of a OR when selecting
+   multiple items in the same facets, thus reducing the scope of the
```
Comment (pronguen) [inline RELEASE-NOTES.rst:65]:
```suggestion
   multiple items in the same facet, thus reducing the scope of the
```

File: RELEASE-NOTES.rst [hunk 9]
```diff
@@ -18,6 +18,223 @@
...
+   filter instead of expanding it.
+-  Improves the ES mapping to enhance the search quality.
+-  Adds a RERO ILS custom analyzer as a default analyzer.
+-  Improves languages analyzer with ``unicode`` capabilities (oe, œ, ue,
+   ü, etc.). Uses a new ES docker image with ``icu`` plugins for this
```
Comment (pronguen) [inline RELEASE-NOTES.rst:70]:
```suggestion
   ü, etc.). Uses a new ES docker image with ``icu`` plugins for these
```

File: RELEASE-NOTES.rst [hunk 10]
```diff
@@ -18,6 +18,223 @@
...
+-  Checks that there is only one pickup location by library.
+-  Processes notification asynchronously, which is more scalable and
+   reliable.
+-  Implements *paging request to closed stack* functionality, needed and
+   developed by
+   `UCLouvain <https://uclouvain.be/en/libraries/about.html>`__. It
+   allows to restrict, for a specific location, the available pickup
+   locations, or even to disable the request option for the whole
+   location. It also allows to set a manager for these paging requests,
+   who is notified.
```
Comment (pronguen) [inline RELEASE-NOTES.rst:85]:
```suggestion
   who receives printable e-mail notifications for each request.
```

File: RELEASE-NOTES.rst [hunk 11]
```diff
@@ -18,6 +18,223 @@
...
+
+   -  Updates the location detailed view with new “paging request”
+      fields.
+   -  Updates the location brief view to identify the “closed stacks”.
+      Adds tool tip message when the location can’t be deleted. Updates
```
Comment (pronguen) [inline RELEASE-NOTES.rst:90]:
```suggestion
      Adds tooltip message when the location can’t be deleted. Updates
```

File: RELEASE-NOTES.rst [hunk 12]
```diff
@@ -18,6 +18,223 @@
...
+   issue*, in which the librarian should be able to add an item without
+   having to attribute a barcode and a call number to it.
+-  Creates a separate table for each resource in the database. Import
+   and export of a resource is easier and access to records faster in
+   big datasets.
+-  Updates JSON schema to draft 7.
+-  Adds methods to ensure PID’s are actually unique.
+-  Improves JSON schema and ES mapping of the patron transaction event.
+-  Implements physical description in the new data model: extent,
+   duration, format, illustrations colors and physical details.
```
Comment (pronguen) [inline RELEASE-NOTES.rst:116]:
```suggestion
   duration, format, illustrations, colors and physical details.
```

File: RELEASE-NOTES.rst [hunk 13]
```diff
@@ -18,6 +18,223 @@
...
+   to provide multilingual capabilities, but the source authority files
+   ID’s are much more stable.
+-  Updates the document editor to propose person authority from the
+   IdRef and GND authority record from the MEF server.
+
+User management
+~~~~~~~~~~~~~~~
+
+-  Adds manual blocking of patrons by the librarian. Blocked patrons
+   can’t check out document, place request on document, and are informed
```
Comment (pronguen) [inline RELEASE-NOTES.rst:130]:
```suggestion
   can’t check out documents or place requests on documents, and are informed
```

File: RELEASE-NOTES.rst [hunk 14]
```diff
@@ -18,6 +18,223 @@
...
+   the ``SearchSerializer``.
+-  Improves the way ``rero-ils-ui`` is getting permissions, through the
+   permission API instead of a search query.
+
+Documentation
+~~~~~~~~~~~~~
+
+-  Adds an informative ``README.rst``, addressed to the general public
+   and developers, explaining the context of the project, what it does
+   and will do, where to find documentation on how to develop, install
```
Comment (pronguen) [inline RELEASE-NOTES.rst:159]:
```suggestion
   and is going to do, where to find documentation on how to develop, install
```

File: RELEASE-NOTES.rst [hunk 15]
```diff
@@ -18,6 +18,223 @@
...
+-  Improves the handling of scheduled task with the use of REDIS
+   scheduler backend, allowing to enable, disable, update, create
+   scheduled tasks dynamically.
+
+Issues
+~~~~~~
+
+-  `#91 <https://github.com/rero/rero-ils/issues/91>`__: The facets
+   behaviour was not as expected. It associated two items with an OR
+   instead of a AND operator.
```
Comment (pronguen) [inline RELEASE-NOTES.rst:213]:
```suggestion
   instead of an AND operator.
```

File: RELEASE-NOTES.rst [hunk 16]
```diff
@@ -18,6 +18,223 @@
...
+-  `#675 <https://github.com/rero/rero-ils/issues/675>`__: A question
+   was raised on how to improve the library custom editor, specifically
+   the opening hour section. It was decided to hide the opening hours
+   when of the closed days.
```
Comment (pronguen) [inline RELEASE-NOTES.rst:217]:
```suggestion
   the opening hours section. It was decided to hide the opening hours
   for closed days.
```

File: RELEASE-NOTES.rst [hunk 17]
```diff
@@ -18,6 +18,223 @@
...
+   population of items in editor selectors was very slow.
+-  `#850 <https://github.com/rero/rero-ils/issues/850>`__: Creation of
+   two records with the same PID is possible.
+-  `#884 <https://github.com/rero/rero-ils/issues/884>`__: Removes the
+   public item detailed view as it is not useful anymore.
+-  `#890 <https://github.com/rero/rero-ils/issues/890>`__: Action
+   realised in circulation should be in past participle, not in the
+   infinitive form.
+-  `#932 <https://github.com/rero/rero-ils/issues/932>`__: Librarian
+   were able to edit item types and patron types, but this actions
```
Comment (pronguen) [inline RELEASE-NOTES.rst:230]:
```suggestion
   were able to edit item types and patron types, but these actions
```

File: RELEASE-NOTES.rst [hunk 18]
```diff
@@ -18,6 +18,223 @@
...
+~~~~
+
+-  Updates the item JSON schema in order to remove requirement on the
+   call number. The item barcode is still required, but can be left
+   empty by the librarian and then is automatically set by the system.
+   This changes are needed to a next to come functionality, *receive an
+   issue*, in which the librarian should be able to add an item without
+   having to attribute a barcode and a call number to it.
+-  Creates a separate table for each resource in the database. Import
+   and export of a resource is easier and access to records faster in
```
Comment (mmo) [inline RELEASE-NOTES.rst:110]:
```suggestion
   and export of a resource are easier and access to records faster in
```

File: RELEASE-NOTES.rst [hunk 19]
```diff
@@ -18,6 +18,223 @@
...
+   big datasets.
+-  Updates JSON schema to draft 7.
+-  Adds methods to ensure PID’s are actually unique.
```
Comment (mmo) [inline RELEASE-NOTES.rst:113]:
```suggestion
-  Adds methods to ensure PIDs are unique.
```

File: RELEASE-NOTES.rst [hunk 20]
```diff
@@ -18,6 +18,223 @@
...
+-  Improves JSON schema and ES mapping of the patron transaction event.
+-  Implements physical description in the new data model: extent,
+   duration, format, illustrations colors and physical details.
+-  Fixes creation, merging and deletion of holdings record for ebooks,
```
Comment (mmo) [inline RELEASE-NOTES.rst:117]:
```suggestion
-  Fixes creation, merging and deletion of holdings records for ebooks,
```

File: RELEASE-NOTES.rst [hunk 21]
```diff
@@ -18,6 +18,223 @@
...
+   during harvesting.
+-  Links document to person authority record through IdRef, GND or RERO
```
Comment (mmo) [inline RELEASE-NOTES.rst:119]:
```suggestion
-  Links documents to person authority records through IdRef, GND or RERO
```

File: RELEASE-NOTES.rst [hunk 22]
```diff
@@ -18,6 +18,223 @@
...
+   ID instead of a MEF record ID. The MEF clustered record is still used
+   to provide multilingual capabilities, but the source authority files
+   ID’s are much more stable.
```
Comment (mmo) [inline RELEASE-NOTES.rst:122]:
```suggestion
   IDs are much more stable.
```

File: RELEASE-NOTES.rst [hunk 23]
```diff
@@ -18,6 +18,223 @@
...
+-  Updates JSON schema to draft 7.
+-  Adds methods to ensure PID’s are actually unique.
+-  Improves JSON schema and ES mapping of the patron transaction event.
+-  Implements physical description in the new data model: extent,
+   duration, format, illustrations colors and physical details.
+-  Fixes creation, merging and deletion of holdings record for ebooks,
+   during harvesting.
+-  Links document to person authority record through IdRef, GND or RERO
+   ID instead of a MEF record ID. The MEF clustered record is still used
+   to provide multilingual capabilities, but the source authority files
```
Comment (mmo) [inline RELEASE-NOTES.rst:121]:
```suggestion
   to provide multilingual capabilities, but the source authority
```

File: RELEASE-NOTES.rst [hunk 24]
```diff
@@ -18,6 +18,223 @@
...
+   ID’s are much more stable.
+-  Updates the document editor to propose person authority from the
```
Comment (mmo) [inline RELEASE-NOTES.rst:123]:
```suggestion
-  Updates the document editor to propose person authorities from
```

File: RELEASE-NOTES.rst [hunk 25]
```diff
@@ -18,6 +18,223 @@
...
+   IdRef and GND authority record from the MEF server.
```
Comment (mmo) [inline RELEASE-NOTES.rst:124]:
```suggestion
   IdRef and GND records in the MEF server.
```

File: RELEASE-NOTES.rst (hunk 13 above)
Comment (mmo) [inline RELEASE-NOTES.rst:130]:
```suggestion
   can’t check out or place requests on documents, and are informed
```

File: RELEASE-NOTES.rst [hunk 26]
```diff
@@ -18,6 +18,223 @@
...
+   the email does not already exist. A validation message is displayed
+   in the editor.
+-  Exposes PO based translations in JSON through a new API HTTP endpoint
+   to the angular application (``rero-ils-ui``), to avoid translating
+   the same strings in both projects. On the ``rero-ils-ui`` side, the
+   translation mechanism is updated to consume the exposed JSON file
+   translations.
+-  Adds ``create`` to the permission API and removes permissions from
+   the ``SearchSerializer``.
+-  Improves the way ``rero-ils-ui`` is getting permissions, through the
```
Comment (mmo) [inline RELEASE-NOTES.rst:151]:
```suggestion
-  Improves the way ``rero-ils-ui`` gets permissions, through the
```

File: RELEASE-NOTES.rst [hunk 27]
```diff
@@ -18,6 +18,223 @@
...
+   permission API instead of a search query.
+
+Documentation
+~~~~~~~~~~~~~
+
+-  Adds an informative ``README.rst``, addressed to the general public
+   and developers, explaining the context of the project, what it does
+   and will do, where to find documentation on how to develop, install
+   or contribute to RERO ILS.
+-  Updates ``rero-ils-ui`` issue template in order to remind user at
```
Comment (mmo) [inline RELEASE-NOTES.rst:161]:
```suggestion
-  Updates the ``rero-ils-ui`` issue template in order to remind users
```

File: RELEASE-NOTES.rst [hunk 28]
```diff
@@ -18,6 +18,223 @@
...
+   creation to privilege issue creation in the ``rero-ils`` GitHub
```
Comment (mmo) [inline RELEASE-NOTES.rst:162]:
```suggestion
   to privilege issue creation in the ``rero-ils`` GitHub
```

File: RELEASE-NOTES.rst [hunk 29]
```diff
@@ -18,6 +18,223 @@
...
+-  Fixes unit tests for item barcode automatically generated (prefixed
+   with “f-”), to ensure that the time stamp of the generated barcode
+   equals the ``sysdate`` time stamp.
+-  Tests the ES simple query with provided search use case in different
+   languages.
+
+``rero-ils-ui``
+~~~~~~~~~~~~~~~
+
+-  Rewrites ``MainTitleService`` as a pipe to ease its use into
```
Comment (mmo) [inline RELEASE-NOTES.rst:181]:
```suggestion
-  Rewrites ``MainTitleService`` as a pipe to ease its use in
```

File: RELEASE-NOTES.rst [hunk 30]
```diff
@@ -18,6 +18,223 @@
...
+   components.
+-  Fixes test component name to be coherent with component name.
```
Comment (mmo) [inline RELEASE-NOTES.rst:183]:
```suggestion
-  Fixes the test component name to be coherent with the component name.
```

File: RELEASE-NOTES.rst [hunk 31]
```diff
@@ -18,6 +18,223 @@
...
+-  Rewrites tests to limit imports and declarations.
+-  Fixes private attribute names that were missing the leading
+   underscore.
+
+Instance
+~~~~~~~~
+
+-  Moves from ``pipenv`` to ``poetry`` to improve dependencies
```
Comment (mmo) [inline RELEASE-NOTES.rst:191]:
```suggestion
-  Moves from ``pipenv`` to ``poetry`` to improve dependency
```

File: RELEASE-NOTES.rst [hunk 32]
```diff
@@ -18,6 +18,223 @@
...
+   issue fixed with the item view and the tests.
+-  Removes a bad hack with ``appnope`` package for Mac OSX.
+-  Removes ``setuptools`` manifest which is not used.
+-  Configures ``celery`` to load ``.env`` and ``.flaskenv`` files.
+-  Adds ``invenio-logging`` Sentry extensions.
+-  Removes ``pipenv`` environment variables from the ``setup`` script.
+-  Fixes an error when interrupting the ``server`` script, resulting in
+   processes still running, after the move from ``pipenv`` to
+   ``poetry``.
+-  Sets the ``setup`` script to skip ebooks harvesting by default.
```
Comment (mmo) [inline RELEASE-NOTES.rst:203]:
```suggestion
-  Sets the ``setup`` script to skip ebook harvesting by default.
```

File: RELEASE-NOTES.rst [hunk 33]
```diff
@@ -18,6 +18,223 @@
...
+-  Improves the handling of scheduled task with the use of REDIS
```
Comment (mmo) [inline RELEASE-NOTES.rst:204]:
```suggestion
-  Improves the handling of scheduled tasks with the use of REDIS
```

File: RELEASE-NOTES.rst (hunk 15 above)
Comment (mmo) [inline RELEASE-NOTES.rst:213]:
```suggestion
   instead of an AND operator.
```

File: RELEASE-NOTES.rst (hunk 16 above)
Comment (mmo) [inline RELEASE-NOTES.rst:217]:
```suggestion
   for closed days.
```

File: RELEASE-NOTES.rst [hunk 34]
```diff
@@ -18,6 +18,223 @@
...
+   when of the closed days.
+-  `#755 <https://github.com/rero/rero-ils/issues/755>`__: The search
+   failed with a query containing brackets ``[]``.
+-  `#819 <https://github.com/rero/rero-ils/issues/819>`__: The
+   population of items in editor selectors was very slow.
+-  `#850 <https://github.com/rero/rero-ils/issues/850>`__: Creation of
+   two records with the same PID is possible.
+-  `#884 <https://github.com/rero/rero-ils/issues/884>`__: Removes the
+   public item detailed view as it is not useful anymore.
+-  `#890 <https://github.com/rero/rero-ils/issues/890>`__: Action
```
Comment (mmo) [inline RELEASE-NOTES.rst:226]:
```suggestion
-  `#890 <https://github.com/rero/rero-ils/issues/890>`__: Actions
```

File: RELEASE-NOTES.rst [hunk 35]
```diff
@@ -18,6 +18,223 @@
...
+   realised in circulation should be in past participle, not in the
+   infinitive form.
+-  `#932 <https://github.com/rero/rero-ils/issues/932>`__: Librarian
```
Comment (mmo) [inline RELEASE-NOTES.rst:229]:
```suggestion
-  `#932 <https://github.com/rero/rero-ils/issues/932>`__: Librarians
```

File: RELEASE-NOTES.rst (hunk 17 above)
Comment (mmo) [inline RELEASE-NOTES.rst:230]:
```suggestion
   were able to edit item types and patron types, but these actions
```

File: RELEASE-NOTES.rst [hunk 36]
```diff
@@ -18,6 +18,223 @@
...
+   were able to edit item types and patron types, but this actions
+   should be reserved to system librarians.
+-  `#934 <https://github.com/rero/rero-ils/issues/934>`__: Searching for
+   patrons in a large data set should rely on a good ranking, in order
+   to get an exact match on the top of the results page.
+-  `#1000 <https://github.com/rero/rero-ils/issues/1000>`__: A test on
+   documents API was not raising exceptions, resulting in a failed test.
```
Comment (mmo) [inline RELEASE-NOTES.rst:236]:
```suggestion
   the document API was not raising exceptions, resulting in a failed test.
```

File: RELEASE-NOTES.rst [hunk 37]
```diff
@@ -18,6 +18,223 @@
...
+   ü, etc.). Uses a new ES docker image with ``icu`` plugins for this
+   functionalities.
+
+Circulation
+~~~~~~~~~~~
+
+-  Checks that there is only one pickup location by library.
+-  Processes notification asynchronously, which is more scalable and
+   reliable.
+-  Implements *paging request to closed stack* functionality, needed and
```
[the remark this answers is not in the document]
Comment (iGormilhit) [reply RELEASE-NOTES.rst:79]: No it's not, I'm amending it.

File: RELEASE-NOTES.rst [hunk 38]
```diff
@@ -18,6 +18,219 @@
...
+   “description” tab with extended document metadata and, in the professional
+   view, an “online” tab when the resource is available through an hyperlink.
+   On top of these tabs are displayed the main metadata of the document with
+   the cover thumbnail.
+
+Public interface
+^^^^^^^^^^^^^^^^
+
+-  Removes the item detailed public view which is useless, all relevant
+   information being on the public  document detailed view.
```
Comment (jma) [inline RELEASE-NOTES.rst:41]: two spaces, all relevant informationS?

File: RELEASE-NOTES.rst [hunk 39]
```diff
@@ -18,6 +18,219 @@
...
+   The API requests are still done through the ES query string, as complex
+   queries are needed to populate the user interface. A new HTTP query optional
+   parameter is added to identify the simple query: ``&simple=1``.
+-  Sets the same AND boolean operator instead of OR when selecting
+   multiple items in the same facet, thus reducing the scope of the
+   filter instead of expanding it.
+-  Improves the ES mapping to enhance the search quality.
+-  Adds a RERO ILS custom analyzer as the default analyzer.
+-  Improves language analyzer with ``unicode`` capabilities (oe, œ, ue, ü,
+   etc.). Uses a new ES docker image with ``icu`` plugins for these
```
Comment (jma) [inline RELEASE-NOTES.rst:67]: You can add a link to the plugin if you want.

File: RELEASE-NOTES.rst [hunk 40]
```diff
@@ -18,6 +18,219 @@
...
+
+Instance
+~~~~~~~~
+
+-  Moves from ``pipenv`` to ``poetry`` to improve dependency
+   management. Uses ``python-dotenv`` to load ``.env`` and ``.flaskenv``
+   files. This allowed to upgrade ``werkzeug`` which resulted in an
+   issue fixed with the item view and the tests.
+-  Removes a bad hack with ``appnope`` package for Mac OSX.
+-  Removes ``setuptools`` manifest which is not used.
```
Comment (jma) [inline RELEASE-NOTES.rst:192]: is not used anymore?

File: RELEASE-NOTES.rst [hunk 41]
```diff
@@ -18,6 +18,219 @@
...
+-  Configures ``celery`` to load ``.env`` and ``.flaskenv`` files.
+-  Adds ``invenio-logging`` Sentry extensions.
+-  Removes ``pipenv`` environment variables from the ``setup`` script.
+-  Fixes an error when interrupting the ``server`` script, resulting in
+   processes still running, after the move from ``pipenv`` to
+   ``poetry``.
+-  Sets the ``setup`` script to skip ebook harvesting by default.
```
Comment (jma) [inline RELEASE-NOTES.rst:199]: This was already the case in the previous release.

**Pull request merged**